packages/xpetra/src/BlockedCrsMatrix/Xpetra_BlockedCrsMatrix_def.hpp

```diff
@@ -68,7 +68,7 @@ BlockedCrsMatrix<Scalar, LocalOrdinal, GlobalOrdinal, Node>::BlockedCrsMatrix(co
   // add CrsMatrix objects in row,column order
   for (size_t r = 0; r < Rows(); ++r)
     for (size_t c = 0; c < Cols(); ++c)
-      blocks_.push_back(MatrixFactory::Build(getRangeMap(r, bRangeThyraMode_), numEntriesPerRow));
+      blocks_.push_back(Xpetra::MatrixFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build(getRangeMap(r, bRangeThyraMode_), numEntriesPerRow));
```

Comment:
Why is this necessary? We include a header with short names. And this did compile just fine before, no?